CHANGELOG.md

```diff
@@ -13,8 +13,12 @@ Possible log types:
 
 ### Unreleased
 
+...
+
+### v0.6.0 (2018-01-25)
+
 - [changed] Minimal required Rust version 1.20.0 (#68)
-- [changed] Update `log` crate to version 0.4
+- [changed] Update `log` crate to version 0.4 (#70)
 - [changed] Use an enum (`IssueReportChannel`) for the `issue_report_channels`
   field (#4)
```

Comment:
This links to the issue, but don't we usually link to the PR?

Reply:
I think it doesn't really matter since they are both linked :) Also, before creating the PR there is no pull request number yet 😉

Reply:
Also why does GitHub create new issues with such low numbers? Did they somehow get missing? Or was this due to our abuse of changing issues to PRs with the CLI tool?

Reply:
Hm, I think that issue is actually that old.